synced_bcf_reader.c

```diff
@@ -75,7 +75,11 @@ char *bcf_sr_strerror(int errnum)
             return "no BGZF EOF marker; file may be truncated"; break;
         case no_memory:
             return "Out of memory"; break;
-        default: return "";
+        case vcf_parse_error:
+            return "VCF parse error"; break;
+        case bcf_read_error:
+            return "BCF read error"; break;
+        default: return ""; 
```

Comment:
Trailing spaces added on the `default` line.  I guess with this rebasing this is otherwise ready to be merged?